### DISCUSSION TITLE
Usage of `swr` option in `routeRules`

### DISCUSSION THREAD

**Discussion by mogi86** [nuxt/nuxt#18477] [Questions]
I specified the `swr` option in `routeRules` in `nuxt.config.ts`, but it did not work as expected. I specified the `swr` option for a page that displays an API response that takes 5 seconds.
I expected that the cache would be used on the second access and the page would be displayed immediately. However, the API was actually called on the second access, and the page took 5 seconds to display. I confirmed on local pc by using `yarn dev` or `yarn build & yarn preview`.
What are the correct `routeRules` to get the expected behavior?

URL path with `swr` option: `/swr`
Reproduction: https://github.com/mogi86/study-nuxt3-v2/blob/v0.0.10/nuxt.config.ts

**Comment by TheAlexLichter** [comment]
`swr` is only available on Netlify and Vercel right now. Follow e.g. https://github.com/unjs/nitro/issues/603 for more.

**Reply by mogi86** [reply]
Thanks. Is it impossible to use cache including other ways than swr?

**Reply by TheAlexLichter** [reply]
Right now there is no implementation for the node preset for swr or static. But I'm pretty sure that a custom implementation would be possible (maybe even with a PR to nitro in the end? 😋).

**Reply by mogi86** [reply]
I got it. thank you!
